src/storage/txn/scheduler.rs

```diff
@@ -772,6 +779,15 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
                 .process_write(snapshot, context)
                 .map_err(StorageError::from)
         };
+
+        sample.add_write_bytes(write_bytes);
+        let quota_delay = quota_limiter.async_consume(sample).await;
```

Comment:
As I commentted before, the write bytes can be a bit inaccurate due to error requests or in-memory pessimistic locks. Let's leave a todo comment here.